.prettierignore

```diff
@@ -1,7 +1,7 @@
 .vscode
 container_deployment
 container-scripts
-MATScommon
+# MATScommon
```

Comment:
A small nit - could you delete this instead of commenting it out? I always forget to go back and delete commented-out code later.

Reply:
done and pushed

Reply:
Thanks, Gopa!